### PR TITLE
Optimize copywriting (text content)

### PR DESCRIPTION
**Pull request by mocklee**
Problem
=======
I want to understand the mission, goals, and opportunities CruzHacks evangelizes. 

Solution
========
Color in some more details from the initial copywriting

### CODE REVIEW

File: src/views/Home/props/prizes.js [hunk 1]
```diff
@@ -10,19 +10,19 @@ export const topics = [
     title: "Health Hacks",
     icon: <HealthHacksIcon />,
```
Comment (ilopezro) [inline src/views/Home/props/prizes.js:11]: Yo this is actually super dope! Have a component in as a prop for the JSON object! Didn't catch this in the #97 PR.

**Review by ilopezro — APPROVED** [review]
This looks really good to me! 🚢

File: src/stylesheets/_typography.scss [hunk 2]
```diff
@@ -1,4 +1,5 @@
 @use "~stylesheets/responsiveness";
+@use "~stylesheets/variables";
```
Comment (ilopezro) [inline src/stylesheets/_typography.scss:2]: I'm wondering what this does though 🤔

Comment (mocklee) [reply src/stylesheets/_typography.scss]: still cleaning up a bit!

**Pull request merged**